socorro/unittest/schemas/test_socorro_data_schemas.py

```diff
@@ -0,0 +1,436 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from click.testing import CliRunner
+import jsonschema
+import pytest
+
+from socorro.lib.libsocorrodataschema import (
+    FlattenKeys,
+    permissions_transform_function,
+    transform_schema,
+)
+from socorro.schemas import get_file_content
+from socorro.schemas.validate_processed_crash import validate_and_test
+
+
+def test_validate_socorro_data_schema():
+    """Assert that socorro-data-1-0-0.schema.yaml is valid jsonschema"""
+    schema = get_file_content("socorro-data-1-0-0.schema.yaml")
+    jsonschema.Draft7Validator.check_schema(schema)
```

Comment:
`socorro-data-1-0-0.schema.yaml` is a Draft 7 schema because I needed format or something. I forget what it was.